Makefile.in

```diff
@@ -489,16 +489,21 @@ fmt:
 	  $$(find middle_end/flambda2 \
 	    \( -name "*.ml" -or -name "*.mli" \) \
 	    -and \! \( -name "flambda_parser.*" -or -name "flambda_lex.*" \))
+	ocamlformat -i \
+	  $$(find backend/cfg \
+	    \( -name "*.ml" -or -name "*.mli" \))
 
 .PHONY: check-fmt
 check-fmt:
-	if [ "$$(git status --porcelain middle_end/)" != "" ]; then \
+	if [ "$$(git status --porcelain middle_end/)" != "" ] || \
+           [ "$$(git status --porcelain backend/)" != "" ]; then \
```

Comment:
This should say `backend/cfg/`, no?  Same below.

Reply:
Fixed both cfg and flamdba2 case here and below.